models/packages/package.go

```diff
@@ -128,14 +128,21 @@ func TryInsertPackage(ctx context.Context, p *Package) (*Package, error) {
 		LowerName: p.LowerName,
 	}
 
-	has, err := e.Get(key)
+	has, err := e.Exist(key)
```

Comment:
I think this is wrong because `Exist` does not populate the object with the content from the database.